Comp2024/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -56,15 +56,15 @@ public class Intake extends SubsystemBase
   private static final double       kRollerSpeedExpel   = -0.25;
 
   private final double              kLigament2dOffset   = 0.0; // Offset from mechanism root for wrist ligament
-  public static final double        kAngleMin           = -2.0;
+  public static final double        kAngleMin           = -90.0;
   public static final double        kAngleStow          = 0.0;
```

Comment:
Delete/edit all angles down to: Min, retracted, handoff, deployed, max values. We will have the two end stops (min/max) and three functional positions (retracted-shooter, handoff to feeder, and deployed to intake).